### ISSUE TITLE
[CDP] Improve alert messaging for better screenreader experience

### ISSUE THREAD

**Issue by josephrlee** [department-of-veterans-affairs/va.gov-team#88882]
## Issue Description

Continuation of #75547

> ## User Story or Problem Statement:
> As a screen reader user, it is easier to navigate error messages when they contain an action item I.E. "return to my VA debt," and when focus is placed on said action item when the error is triggered. 
>
> ## Details
> In the event that there are no debt is associated with a user account an error message appears on the screen. The screen reader announces the message in full, and keyboard focus is placed on the h1. The user can return to the previous page by navigating up to the breadcrumb.
User experience can be improved by including a "Return" link within the error message and moving focus to it upon the triggering of the message. 

Update alerts in these components from these pages:

- [ComboAlerts.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/combined-debt-portal/combined/components/ComboAlerts.jsx) that appears on combined debt portal [OverviewPage.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/combined-debt-portal/combined/containers/OverviewPage.jsx)
- Any [Alerts.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/combined-debt-portal/debt-letters/components/Alerts.jsx) that appears on [DebtLettersSummary.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/combined-debt-portal/debt-letters/containers/DebtLettersSummary.jsx)
- [alert-messages.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/combined-debt-portal/combined/utils/alert-messages.jsx) on the copay [OverviewPage.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/combined-debt-portal/medical-copays/containers/OverviewPage.jsx)

These are the pages where the above alerts display:

- https://staging.va.gov/manage-va-debt/summary, [figma pages](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=0-6350&t=kvf1BzcC6lSNRQeE-11).
*Note: On the Debt and Bills summary page, users cannot be sent back to Manage My Debt because of a redirect. The return link on the Debt and Bills summary page should read _Return to VA.gov_ and link to  _VA.gov_.
- https://staging.va.gov/manage-va-debt/summary/copay-balances, [figma pages](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=0-7495&t=kvf1BzcC6lSNRQeE-11)
- https://staging.va.gov/manage-va-debt/summary/debt-balances, [figma pages](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=0-11158&t=kvf1BzcC6lSNRQeE-11)
*Note: On Overpayment summary page and Copay summary page, the return link should read _View all your VA debt and bills_ and link to the Debt and Bills summary page, _va.gov/manage-va-debt/summary/_.
---
## Tasks
- [x] Update alerts based off updated designs in figma

## Acceptance Criteria
- [ ] All alerts have been updated based off of designs in figma
- [ ] Given there is an error message displayed on the screen
  - When the error message is triggered
  - Then the error message should include a "Return" link
  - And the focus should be moved to the "Return" link
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by HeatherRienks** [comment]
Per Planning, this will roll over and 2pts of effort remain.

**Comment by josephrlee** [comment]
I paired with @Dr-Pongo and was able to complete the majority of this effort. This will roll over into the next sprint, though, but it will be able to be closed out soon! 

cc: @HeatherRienks @Tom-Davis @maxx1128
